build.php

```diff
@@ -19,13 +19,20 @@
     exit(0);
 }
 
+$dbNames = array_map(function ($info) {
+    return $info['db_name'];
+}, $dbs);
+
+$indexRoots = getIndexRoots($wbapi, $wbitem, $dbNames);
 foreach ($dbs as $lang => $info) {
-    $dsn = $info['dsn'];
-    $indexNs = $info['index_ns'];
-    $indexRoot = $info['index_root'];
-    $catLabel = $info['cat_label'];
-    $pdo = new PDO($dsn, $dbuser, $dbpass);
-    buildOneLang( $pdo, $lang, $indexRoot, $catLabel, $indexNs );
+    $dbName = $info['db_name'];
+    if (isset($indexRoots[$dbName])) {
+        $dsn = "mysql:dbname={$dbName}_p;host=$dbName.labsdb";
```

Comment:
I didn't want to construct the DSN like this, only because it makes it harder for local testing whilst in development.